ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1010,6 +1071,8 @@ const RuntimePanel = fp.flow(
   const pdExists = !!persistentDisk;
   const pdSize = pdExists ? persistentDisk.size : defaultDiskSize;
   const initialAutopauseThreshold = existingRuntime.autopauseThreshold || DEFAULT_AUTOPAUSE_THRESHOLD_MINUTES;
+  const gpuConfig = !!gceConfig && !!gceConfig.gpuConfig ? gceConfig.gpuConfig : null;
```

Comment:
You don't need the `!!` here. It's only necessary if you actually explicitly want to get a boolean as the output of the expression. In this case, you're just evaluating the overall predicate - the expression does not return `!!gceConfig`, instead if returns the ternary result.